supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -67,3 +67,834 @@ For documentation questions, contact rhel-docs@redhat.com.
+==== image:images/yes.png[yes] Certificate Authorities (noun)
+*Description*: An entity that issues digital certificates. In {RH} {IPA}, the primary CA is `ipa`, the IdM CA. The `ipa` CA certificate is one of the following types:
+--
+* Self-signed. In this case, the `ipa` CA is the root CA.
+* Externally signed. In this case, the `ipa` CA is subordinated to the external CA.
+--
+In IdM, you can also create multiple *sub-CAs*. Sub-CAs are IdM CAs whose certificates are one of the following types:
+
+* Signed by the `ipa` CA.
+* Signed by any of the intermediate CAs between itself and `ipa` CA. The certificate of a sub-CA cannot be self-signed.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:certificate[certificate]
+
+[discrete]
+[[commit]]
+==== image:images/yes.png[yes] commit (noun)
+*Description*: A release or image version of the operating system. Image Builder generates an ostree commit for RHEL for Edge images. You can use these images to install or update RHEL on Edge servers.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ostree[ostree]
+
+[discrete]
+[[compose]]
+==== image:images/yes.png[yes] compose (noun)
+*Description*: Composes are individual builds of a system image, based on a particular version of a particular blueprint. Compose as a term refers to the system image, the logs from its creation, inputs, metadata, and the process itself.
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:blueprint[blueprint]
+
+[discrete]
+[[control-node]]
+==== image:images/yes.png[yes] control node (noun)
+*Description*: ny machine with Ansible installed. You can run commands and playbooks, invoking /usr/bin/ansible or /usr/bin/ansible-playbook, from any control node. You can use any computer that has Python installed on it as a control node - laptops, shared desktops, and servers can all run Ansible. However, you cannot use a Windows machine as a control node. You can have multiple control nodes.
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ansible-playbook[ansible-playbook], xref:ansible-playbook-sysroles[ansible-playbook-sysroles]
+
+[discrete]
+[[cross-forest-trust]]
+==== image:images/yes.png[yes] cross-forest trust (noun)
+*Description*: A trust establishes an access relationship between two Kerberos realms, allowing users and services in one domain to access resources in another domain.
+With a cross-forest trust between an Active Directory (AD) forest root domain and an IdM domain, users from the AD forest domains can interact with Linux machines and services from the IdM domain. From the perspective of AD, Identity Management represents a separate AD forest with a single AD domain. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/{ProductNumberLink}/html/installing_identity_management/installing-trust-between-idm-and-ad_installing-identity-management#how-the-trust-works_installing-trust-between-idm-and-ad[How the trust works].
```

Comment:
Same issue with the attribute here for {ProductNumberLink}, so this link won't work. 

Can you do a search for "{" in your PR, and update all of these attributes to their correct hard coded values?

Reply:
+1